pages/collection/[id]/play/index.js

```diff
@@ -234,26 +238,26 @@ export default function PlayMode({ collections, flashCards }) {
               )}
             </StyledCardContainer>
             <StyledFooter>
-              <button onClick={handleNext}>❌</button>
+              <StyledX onClick={handleNext}></StyledX>
```

Comment:
I would argue that this should be a button as clicking this element triggers an action. I would use a button and place the icon inside the button. Additionally, I advise to add an aria-label if there is no text label.